### PR TITLE
2.5: REST & API Endpoints

### PR DESCRIPTION
**Pull request by Thom187**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@ node_modules/
 .DS_Store/
 .DS_Store
 log.txt
-log.txt
+.DS_Store
```
Comment (oxenprogrammer) [inline .gitignore:5]: Could you please explain why we have a couple of `DS_Store` instances in this file.

File: index.js [hunk 2]
```diff
@@ -1,18 +1,128 @@
...
+// Return/READ data about a director by name
+app.get('/movies/directors/:directorName', (req, res) => {
+  const { directorName } = req.params;
+  const director = movies.find( movie => movie.Director.Name === directorName ).Director;
+
+  if (director) {
+    res.status(200).json(director);
+  } else {
+    res.status(400).send('Could not find that director.')
+  }
```
Comment (oxenprogrammer) [inline index.js:125]: 👍🏾

**Review by oxenprogrammer — APPROVED** [review]
Well done Tom.

File: .gitignore (hunk 1 above)
Comment (Thom187) [reply .gitignore:5]: Not really. I guess i commited it once and tried to ignore afterwards

**Pull request merged**